nss-cohort-36/keahua-arboretum-net-surfers
# Rb style headers

# Description

Making the menus look pretty with the header and wrappers

## Type of change


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout rb-style-headers`
4. Check functionality in every menu and make sure menus 1-4 all stay wrapped with `---===***===---`

# Checklist:
